stackrox-container-image-scanner/src/main/resources/com/stackrox/jenkins/plugins/StackroxBuilder/config.properties

```diff
@@ -6,3 +6,4 @@ ValidateConnectionTitle=Validate connection
 EnableTLSVerification=Enable TLS verification
 CACertificate=CA Certificate
 ImageNames=Image Names
+ClusterTitle=Cluster (Optional)
```

Comment:
Seeing as we don't add the `(optional)` in the title for other params that are optional (e.g. CA Certificate), let's skip it here and be consistent.

Reply:
https://github.com/stackrox/jenkins-plugin/pull/268/commits/157be18de6c0ca24d6f43a0aefe183f7445e4d08